include/rocksdb/comparator.h

```diff
@@ -134,16 +162,18 @@ class Comparator : public Customizable, public CompareInterface {
   // In the future, we can extend Comparator so that subclasses can specify
   // both largest and smallest timestamps.
   virtual int CompareTimestamp(const Slice& /*ts1*/,
-                               const Slice& /*ts2*/) const {
+                               const Slice& /*ts2*/) const override {
```

Comment:
Just override, not virtual+override.  

This same comment applies to many of the functions below.